examples/usecases/incremental-training-with-layer-freezing.ipynb

```diff
@@ -0,0 +1,883 @@
+{
```

Comment:
<p>I would write it clear, that this is a new scenario. </p><p> </p><p>"Let's consider a new situation..." it is unrelated to the situation above and we train on data_1 again?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/models/pull/809/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>